src/utils/config.py

```diff
@@ -8,27 +8,37 @@
 
 ENVIRONMENT = os.getenv("ENVIRONMENT", ENV_LOCAL)
 
-PREFECT_PROJECT_NAME = os.getenv('PREFECT_PROJECT_NAME', PROJECT_LOCAL)
-PREFECT_S3_RESULT_BUCKET = os.getenv('PREFECT_S3_RESULT_BUCKET')
+PREFECT_PROJECT_NAME = os.getenv("PREFECT_PROJECT_NAME", PROJECT_LOCAL)
+PREFECT_S3_RESULT_BUCKET = os.getenv("PREFECT_S3_RESULT_BUCKET")
 
-GCS_BUCKET = 'pocket-prefect-stage-prod' if ENVIRONMENT == ENV_PROD else 'pocket-prefect-stage-dev'
+GCS_BUCKET = (
+    "pocket-prefect-stage-prod"
+    if ENVIRONMENT == ENV_PROD
+    else "pocket-prefect-stage-dev"
+)
 GCS_PATH_DEFAULT = ENVIRONMENT
-GCS_PATH = os.getenv('GCS_PATH', GCS_PATH_DEFAULT)
-
-SNOWFLAKE_DEV_SCHEMA_DEFAULT = 'DEV_DATA_ENGINEERING'
-SNOWFLAKE_DEV_SCHEMA = os.getenv('SNOWFLAKE_DEV_SCHEMA', SNOWFLAKE_DEV_SCHEMA_DEFAULT)
-SNOWFLAKE_STAGE = 'prefect.public.prefect_gcs_stage_parq_prod' if ENVIRONMENT == ENV_PROD else 'development.public.prefect_gcs_stage_parq_dev'
-SNOWFLAKE_DB = 'prefect' if ENVIRONMENT == ENV_PROD else 'development'
-SNOWFLAKE_MOZILLA_SCHEMA = 'mozilla' if ENVIRONMENT == ENV_PROD else SNOWFLAKE_DEV_SCHEMA
-SNOWFLAKE_ANALYTICS_DATABASE = 'ANALYTICS'
+GCS_PATH = os.getenv("GCS_PATH", GCS_PATH_DEFAULT)
```

Comment:
If you are going to start reformatting all the files you touch perhaps we need to put a linter in place to ensure that we all use the same formatting rules and we don't have excess lines diffed to navigate.

Reply:
I can add black formatting check to the build so that folks have to run black locally to pass the check.  How does that sound.  Python formatting via black is super simple....you just run it  :)

https://github.com/psf/black